pkg/pgmodel/ingestor/copier.go

```diff
@@ -242,7 +277,7 @@ func retryAfterDecompression(conn pgxconn.PgxConn, req copyRequest, minTimeInt i
 
 	metrics.DecompressCalls.Inc()
 	metrics.DecompressEarliest.WithLabelValues(table).Set(float64(minTime.UnixNano()) / 1e9)
-	err, _ := insertSeries(conn, req) // Attempt an insert again.
+	err, _ := insertSeries(context.Background(), conn, req) // Attempt an insert again.
```

Comment:
We should but this particular piece was not of interest since it never happened in benchmark runs so I left it for a future addition. Top level function is traced without going into the retry.

Reply:
If you think this is important to add now, let me know and I'll do it.